pkg/server/admin.go

```diff
@@ -1040,14 +1041,44 @@ func (s *adminServer) Cluster(
 func (s *adminServer) Health(
 	ctx context.Context, req *serverpb.HealthRequest,
 ) (*serverpb.HealthResponse, error) {
+	isLive, err := s.server.nodeLiveness.IsLive(s.server.NodeID())
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "node is not live")
```

Comment:
Why do we return "node is not live" here but the body of the error in the analogous case of the `Ready()` function?

Reply:
Oversight, thanks for catching! Also changed the `Errorf` calls to just `Error`.